### DISCUSSION TITLE
Text Editor - Primeira letra

### DISCUSSION THREAD

**Discussion by Karadarevic** [balta-io/forum#103] [Tira Dúvidas] [answered]
Após a primeira linha, o primeiro caractere de cada linha está sumindo mesmo no arquivo txt. 

![imagem_2022-11-14_145446254](https://user-images.githubusercontent.com/92898328/201731682-2d81b8f2-b20e-4e2b-995e-e16bc3617032.png)
![imagem_2022-11-14_145512630](https://user-images.githubusercontent.com/92898328/201731759-b2efe6ab-5960-4aa8-96af-40890ec49473.png)
![imagem_2022-11-14_145540914](https://user-images.githubusercontent.com/92898328/201731866-6cc6e261-3344-4591-836e-aa9ddc5aeb44.png)

**Comment by andrebaltieri** [comment, marked as answer]
Aqui tem mais detalhes sobre este cenário:
👉 https://github.com/balta-io/2801/discussions/11
